### DISCUSSION TITLE
Proxy server broken? I got  500 error

### DISCUSSION THREAD

**Discussion by puzige** [acheong08/ChatGPT#1084] [General]
```python
  File "main.py", line 27, in chat
    res = await session.chat_with_chatgpt(sentence, user_id=user_id, loop=loop)
  File "/home/bot/chatgpt-bot/engine/session/session.py", line 103, in chat_with_chatgpt
    raise e
  File "/home/bot/chatgpt-bot/engine/session/session.py", line 77, in chat_with_chatgpt
    async for data in bot.chat_gpt_bot.ask(sentence):
  File "/home/bot/chatgpt-bot/engine/venv/lib/python3.8/site-packages/revChatGPT/V1.py", line 816, in ask
    self.__check_response(response)
  File "/home/bot/chatgpt-bot/engine/venv/lib/python3.8/site-packages/revChatGPT/V1.py", line 930, in __check_response
    response.raise_for_status()
  File "/home/bot/chatgpt-bot/engine/venv/lib/python3.8/site-packages/httpx/_models.py", line 749, in raise_for_status
    raise HTTPStatusError(message, request=request, response=self)
httpx.HTTPStatusError: Server error '500 Internal Server Error' for url 'https://chat.duti.tech/api/conversation'
```

Do you guys know what happened?

**Comment by byhirsiz** [comment]
I have the same problem, but I don't know what it is

**Reply by acheong08** [reply]
Seems to be overloaded

**Reply by puzige** [reply]
> Seems to be overloaded

any solution?

**Reply by acheong08** [reply]
- https://github.com/acheong08/ChatGPT/discussions/1081

**Reply by acheong08** [reply]
It should be fixed now. I just had to block a ton of abusing IPs

**Reply by puzige** [reply]
> It should be fixed now. I just had to block a ton of abusing IPs

`export CHATGPT_BASE_URL="https://apps.openai.com/"` I use the offical endpoint, but get a 404 error. 

```
Client error '404 Not Found' for url 'https://apps.openai.com/conversation'
For more information check: https://httpstatuses.com/404
```

**Reply by acheong08** [reply]
`export CHATGPT_BASE_URL="https://apps.openai.com/api/"`

**Reply by puzige** [reply]
still get `Client error '404 Not Found' for url 'https://apps.openai.com/api/conversation'
For more information check: https://httpstatuses.com/404`

**Comment by Sultan790** [comment]
It seems that the error is caused by a server error with status code 500. This means that there is an issue with the server and not with the code itself. You can try running the code again later or contact the server administrator to resolve the issue. There is nothing that can be done with the code to fix this issue.

**Reply by acheong08** [reply]
Server crashed. Load too high
